proto/Makefile

```diff
@@ -0,0 +1,24 @@
+PROTOC_VERSION=3.11.4
```

Comment:
Why do we pull inn this dependency?

Reply:
I don't see this as a dependency per se. We must ensure we git ignore any artifacts that are created in the build process.

Reply:
Conclusion is that proto is a needed dependency currently